docs/apache-airflow-providers-tabular/connections.rst

```diff
@@ -0,0 +1,39 @@
+ .. Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
I _think_ we typically want to have a URI example for connection in the doc. Is that right @mik-laj?